usagov/vote-gov-drupal
# VOTE-852: update permissions for content roles

## Jira ticket (required)
[Insert Jira ticket link](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-852&assignee=609161faf6c0960069bf61b3)

## Description (optional)
Updates permissions for content roles (editor and manager)

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. run lando drush cim

### Testing steps
1. Log in as content editor and content manager
2. Test creating and approving content type
3. Test creating and approving media type
4. Test creating and approving block type
5.  Reference Content Migration Plan guidance document for expected permissions
